packages/subnav/index.js

```diff
@@ -115,7 +119,7 @@ function Subnav({ className, ...restProps }) {
   return (
     <nav
       ref={wrapperRef}
-      className={classNames(s.root, className, {
+      className={classNames(s.root, themeStyles[theme], className, {
```

Comment:
MOST of the theming is done via CSS custom properties - that comes from this class here 👍